integrations/sklearn/Neptune-Scikit-learn.ipynb

```diff
@@ -0,0 +1,2048 @@
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Explore results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You just learned how to log scikit-learn regression summary to Neptune using single function.\n",
+    "\n",
+    "Click on the link that was outputted to the console or [go here](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/charts) to explore an experiment similar to yours. In particular check:\n",
+    "\n",
+    "* [logged regressor parameters](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/details) as properties,\n",
+    "* [logged pickled model](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/artifacts?path=model%2F&file=estimator.skl),\n",
+    "* [logged test predictions](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/artifacts?path=csv%2F&file=test_predictions.csv),\n",
+    "* [logged test scores](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/charts),\n",
+    "* [logged regressor visualizations](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/logs) - look for \"charts_sklearn\",\n",
+    "* [logged metadata](https://ui.neptune.ai/o/shared/org/sklearn-integration/e/SKLEARN-311/details) including git summary info."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Scikit-learn classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 1: Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import load_digits\n",
+    "from sklearn.ensemble import GradientBoostingClassifier\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 2: Create and fit gradient boosting classifier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "gbc = GradientBoostingClassifier(n_estimators=120, learning_rate=0.12, min_samples_split=3, min_samples_leaf=2)\n",
+    "\n",
+    "X, y = load_digits(return_X_y=True)\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20, random_state=28743)\n",
+    "\n",
+    "gbc.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 3: Create Neptune experiment and log regressor summary"
```

Comment:
Again, I'd split it into two, add the part related to token, and the part click on a link to see.

Reply:
ok